go.mod.old

```diff
@@ -0,0 +1,33 @@
+module github.com/newrelic/nri-docker
+
+go 1.16
+
+require (
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
```

Comment:
this should not be here I guess

Reply:
Yes, I will delete it. I don't know why it was there.

Reply:
done!